lib/jasmine-2.6.4/console.js

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Could the Jasmine library have been committed separately from the code that used it? What advantages would this have brought to the readability/changeability of the commit history?